JaymesKat/book-a-meal-frontend
# 160411465: Configure webpack to transpile SCSS

#### What does this PR do?
It adds scripts to handle transpiling of any SCSS files in the project down to CSS 

#### Description of Task to be completed?
- Setup webpack configuration to run SCSS transpiler

#### How should this be manually tested?
After cloning the repository;
- Navigate to the `/src` folder and edit the .scss files and add a style that you would like to see on the application user interface
- Run `npm start` to view changes reflected 

#### Any background context you want to provide?
The project was bootstrapped with `create-react-app` from the Facebook team [source](https://github.com/facebook/create-react-app) which does not support SCSS transpiling out of the box

#### What are the relevant pivotal tracker stories?
[#160411465](https://www.pivotaltracker.com/story/show/160411465)

#### Screenshots
N/A
